x-pack/plugins/security_solution/server/lib/entity_analytics/routes/risk_engine_init_route.ts

```diff
@@ -43,20 +48,31 @@ export const riskEngineInitRoute = (
         const initResult = await riskEngineDataClient.init({
           taskManager,
           namespace: spaceId,
+          isAssetCriticalityEnabled,
+          assetCriticalityDataClient,
+          riskScoreDataClient,
         });
 
-        const initResultResponse = {
+        const initResultResponse: InitRiskEngineResultResponse = {
           risk_engine_enabled: initResult.riskEngineEnabled,
           risk_engine_resources_installed: initResult.riskEngineResourcesInstalled,
           risk_engine_configuration_created: initResult.riskEngineConfigurationCreated,
           legacy_risk_engine_disabled: initResult.legacyRiskEngineDisabled,
           errors: initResult.errors,
         };
 
+        // TODO: after enabling asset criticality by default, we don't need assetCriticalityInstallFailed
```

Comment:
Is this TODO still relevant? It's not clear to me what "enabling criticality by default" entails, whether it's already done, nor where to find that info out.